features/visitor_can_see_articles_on_landing_page.feature

```diff
@@ -1,18 +1,29 @@
+@javascript
+
 Feature: Articles on the landing page
-    As a visitor,
-    In order to choose what article to read,
-    I would like to see the headlines and the first paragraphs of the available articles.
+  As a visitor,
+  In order to choose what article to read,
+  I would like to see the headlines and the first paragraphs of the available articles.
+
+  Background:
+    Given the following categories exist
+      | name        |
+      | Programming |
+      | Education   |
 
-  Scenario: View available articles on the landing page
     Given the following articles exist
-      | header                | subheader                       | body                     |
-      | Top title             | A breaking news item            | Today at craftacademy    |
-      | Other stories         | Another breaking news           | Tomorrow at craftacademy |
-  
+      | header        | subheader             | body                     | category    |
+      | Top title     | A breaking news item  | Today at craftacademy    | Programming |
+      | Other stories | Another breaking news | Tomorrow at craftacademy | Education   |
+
+
+  Scenario: View available articles on the landing page
     When I visit the site
+    And I click on "Programming"
+    Then stop
```

Comment:
We need to remove this - semaphore build not completing.

Reply:
Wow! I missed that. Sorry.